docs/METRICS.md

```diff
@@ -158,6 +158,16 @@ Below is a sample ping for the `badge_toolbar_button` and `visit_supported_site`
   - `'product_card'`: Sends the user to the product page for the given Product Card.
   - `'system_notification'`: Sends the user to the product page for the Price Alert displayed in the notification.
   - `'walmart_link'`: Sends the user to Walmart.
+- `'do_not_track'`: 'true' if the user has requested not to be tracked by websites, content, or advertising; otherwise 'false'.
+- `'privacy_tp'`: The user's tracking protection setting:
```

Comment:
Nit: Why is this `'privacy_tp'` instead of `'tracking_protection'`? I'm going to guess because of string length limits.
If so, perhaps all 3 new `extra_keys` could be `'privacy_*'`? E.g. `'privacy_tp'`, `'privacy_dnt'`, `'privacy_cookies'`

Reply:
> Nit: Why is this `'privacy_tp'` instead of `'tracking_protection'`? I'm going to guess because of string length limits.

Yup.